gimme

```diff
@@ -256,6 +277,28 @@ _try_git() {
 	echo "export GIMME_ENV=\"$(_env_alias "${git_dir}" "${git_env}")\""
 }
 
+_wipe_version() {
+	local env_file="${GIMME_ENV_PREFIX}/go${1}.${GIMME_OS}.${GIMME_ARCH}.env"
+
+	if [[ -s "${env_file}" ]] ; then
+		rm -rf "$(awk -F\" '/GOROOT/ { print $2 }' "${env_file}")"
+		rm -f "${env_file}"
+	fi
+}
+
+_realpath() {
+	[[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
```

Comment:
... and if we want almost fully canonical paths (ala `readlink -f`), we could swap those `pwd` invocations for `pwd -P`, but IMO that's overkill here :+1: